parser/src/python.lalrpop

```diff
@@ -279,14 +279,14 @@ ImportAsAlias<I>: ast::Alias = {
 
 // A name like abc or abc.def.ghi
 DottedName: ast::Identifier = {
-    <n:name> => ast::Identifier::new(n),
-    <n:name> <n2: ("." Identifier)+>  => {
+    <location:@L> <n:name> <end_location:@R> => ast::Identifier::new(n, (location..end_location).into()),
+    <location:@L> <n:name> <n2: ("." Identifier)+> <end_location:@R> => {
         let mut r = n.to_string();
         for x in n2 {
             r.push('.');
             r.push_str(x.1.as_str());
         }
-        ast::Identifier::new(r)
+        ast::Identifier::new(r, (location..end_location).into())
```

Comment:
I'm sorry what... It's weird that `a.b.c` gets parsed as a single identifier instead of a `MemberName` node that has an `object` and `property` node where `object` is a `MemberName` or `Identifier` and `property` is an `Identifier`.